.github/workflows/status-checks.yml

```diff
@@ -0,0 +1,18 @@
+name: Status checks
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+  # Make it possible to trigger the checks manually.
+  workflow_dispatch:
+
+jobs:
+  docker-build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      # We only run the build stage to prevent that the IRMA schemes are downloaded.

```

Comment:
This command can be removed here, because the Angrygames container does not download IRMA schems on build

Reply:
removed